raiden-dapp/src/services/raiden-service.ts

```diff
@@ -338,6 +338,15 @@ export default class RaidenService {
     return this.raiden.config.pfs === undefined;
```

Comment:
I think you may remove the `noPfsSelected` now, you shouldn't need to inspect it, right?

Reply:
I am on it :) I will add an extra commit removing it